### PR TITLE
Adds more keys and edit existing ones

### PR DESCRIPTION
**Pull request by MariusCasvean**

### CODE REVIEW

File: src/i18n/locales/en.json [hunk 1]
```diff
@@ -1299,7 +1301,7 @@
...
...n conformitate cu termenii de soluţionare a litigiilor din Termenii și condiţiile Ease. Dacă vreo prevedere a acestor Termeni este, din orice motiv, considerată ca fiind invalidă sau neaplicabilă, celelalte dispoziţii ale acestor Termeni vor rămâne executorii, iar dispoziţiile invalide sau neexecutabile vor fi considerate modificate astfel încât să fie valabile și executorii în măsura maximă permisă de lege. Orice renunţare sau neîndeplinire a oricărei prevederi a acestor Termeni cu o singură ocazie nu va fi considerată renunţare la nici o altă dispoziţie sau la o asemenea dispoziţie cu altă ocazie. Nu puteţi atribui niciun drept sau obligaţie în conformitate cu acești Termeni, total sau parţial, fără acordul scris al Ease. Ease poate atribui sau transfera în mod liber acești Termeni. Orice atribuire sau transfer cu încălcarea celor menţionate anterior este nulă. Acești Termeni sunt Termenii finali, compleţi și exclusivi între dvs. și Ease cu privire la subiectele de mai sus și înlocuiesc și contopesc orice discuţii între dvs. și Ease cu privire la aceste subiecte. \n\nDacă aveţi întrebări cu privire la Program sau la acești Termeni, vă rugăm să ne contactaţi la support@ease.ro. ",
```
Comment (norbertdemeny13) [inline src/i18n/locales/en.json:1304]: Not sure if `\n\n` works the best here. We would have to use `v-html` to have it working, but that something that can cause some xss attacks. That's why it's not recommended. 

What I would suggest here, if it's possible, to have an array of paragraphs and do a v-for in the component. You can use the power of `i18n template`, see documentation :)

Comment (MariusCasvean) [reply src/i18n/locales/en.json:1304]: not sure if I understand. There are hundreds of places where we use this approach. Why is this one different?

Comment (norbertdemeny13) [reply src/i18n/locales/en.json:1304]: Ignore it. Looks good

**Comment by norbertdemeny13** [comment]
You can merge this

**Pull request merged**